packages/rspack/etc/api.md

```diff
@@ -13141,15 +13149,28 @@ const strictModuleExceptionHandling: z.ZodBoolean;
 
 // @public (undocumented)
 export const SwcCssMinimizerRspackPlugin: {
-    new (options?: any): {
+    new (options?: SwcCssMinimizerRspackPluginOptions | undefined): {
         name: BuiltinPluginName;
+<<<<<<< HEAD
```

Comment:
This needs update, run `pnpm run api-extractor:local`

Reply:
Did the update!